backend/backend/views.py

```diff
@@ -36,4 +40,19 @@ def post(self, request, *args, **kwargs):
         })
 
 
+class Info(APIView):
+
+    def get(self, request, *args, **kwargs):
+        channel_name = get_channel_name(request)
+        channel = settings.LEDGER_CHANNELS[channel_name]
+        return Response({
+            'host': request.META['HTTP_HOST'],
```

Comment:
It's better to use default domain here because of the possibility to use localhost/service/etc

Reply:
fixed in https://github.com/SubstraFoundation/substra-backend/pull/395/commits/829a1bd532f6d7fbe5ffb52586899d39018da8d4